scripts/devtron-reference-helm-charts/reference-chart_3-12-0/templates/hpa.yaml

```diff
@@ -1,6 +1,9 @@
 {{- if $.Values.autoscaling.enabled }}
----
+{{- if semverCompare ">=1.16-0" .Capabilities.KubeVersion.GitVersion }}
```

Comment:
its good practice to write logic in utils and treat `apiVersion`  as variable in template